tests/robustness/Makefile

```diff
@@ -113,10 +121,10 @@ $(GOPATH)/bin/gofail: tools/mod/go.mod tools/mod/go.sum
 	cd /tmp/etcd-v3.5.$*-failpoints/; \
 	  git clone --depth 1 --branch v3.5.$* https://github.com/etcd-io/etcd.git .; \
 	  go get go.etcd.io/gofail@${GOFAIL_VERSION}; \
-	  (cd server; go get go.etcd.io/gofail@${GOFAIL_VERSION}); \
-	  (cd etcdctl; go get go.etcd.io/gofail@${GOFAIL_VERSION}); \
-	  (cd etcdutl; go get go.etcd.io/gofail@${GOFAIL_VERSION}); \
-	  (cd tools/mod; go get go.etcd.io/gofail@${GOFAIL_VERSION}); \
+	  (cd $(REPOSITORY_ROOT)/server; go get go.etcd.io/gofail@${GOFAIL_VERSION}); \
```

Comment:
Don't think those packages should be changed. They are relative to the `cd /tmp/etcd-v3.5.$*-failpoints/`. This applies to all binaries built in `/tmp/etcd-*`

Reply:
Ah, right. Makes sense. Reverting these.

Maybe a dumb question, but why are they using the `gofail` version specified in the parent repository `tools/mod`? Shouldn't they be using the version from the cloned repository?

Reply:
Makes sense, thanks for the explanation.

I'm just making a mental note for myself: We assume the latest version is always backward compatible.